c10/test/util/exception_test.cpp

```diff
@@ -0,0 +1,20 @@
+#include <c10/util/Exception.h>
+#include <gtest/gtest.h>
+#include <stdexcept>
+
+namespace {
+bool throw_func() {
+  throw std::runtime_error("I'm throwing...");
+}
+} // namespace
+
+TEST(ExceptionTest, TORCH_DCHECK) {
+#ifndef NDEBUG
```

Comment:
nit: #ifdef NDEBUG and switch the two branches?

Reply:
switched